test/vm-download

```diff
@@ -102,7 +96,9 @@ def prune():
             targets.append(target)
     for filename in os.listdir(DATA):
         path = os.path.join(DATA, filename)
-        if os.path.isfile(path) and path.endswith(".qcow2") and path not in targets:
+        if not force and os.stat(path).st_mtime > now - testinfra.IMAGE_EXPIRE * 86400:
```

Comment:
Would st_atime be too accommodating here? I'm just thinking about images built for PRs that take a long time to make it in.